pkg/cmd/roachtest/tests/follower_reads.go

```diff
@@ -865,10 +906,12 @@ func getFollowerReadCounts(ctx context.Context, t test.Test, c cluster.Cluster)
 			if err != nil {
 				return err
 			}
-			// follower-reads/mixed-version runs on insecure mode, so we need http.
-			// Tests that do run on secure mode will redirect to https.
-			url := "http://" + adminUIAddrs[0] + "/_status/vars"
-			client := roachtestutil.DefaultHTTPClient(c, t.L())
+			url := "https://" + adminUIAddrs[0] + "/_status/vars"
+			// Make sure to connect to the system tenant in case this test
```

Comment:
Metrics are available per tenant. Some metrics are _system_ metrics -- in this test, we're reading the count of successful follower reads, which is a system metric. Since the mixed-version test changes the default controller on the cluster to be the tenant created by the framework, it means that making the `/_status/vars` request without the `tenant` cookie would get us the metrics for that default tenant (which don't include the follower reads count we're interested in). So in this diff, we're explicitly setting the `tenant` cookie to `system` so that we're able to read system metrics.

Reply:
I see, thanks for explaining! 

> successful follower reads, which is a system metric

Is there a quick and easy way to tell? Or do you just need to understand what the metric is. For follower reads, I see `store` in `cr.store.follower_reads.success_count` so system makes sense, but maybe it just makes sense in this case because you told me it's system 😅 .

Reply:
I don't know if there's a quick and easy way. I think, in terms of the code, it depends on which bucket the metrics are put in:

https://github.com/cockroachdb/cockroach/blob/99e44fb89085b4f81eb7408202847764b0bdd1cc/pkg/server/server.go#L260-L270

But I don't see how you would immediately be able to tell from the metrics alone without having to trace it back to the registry.